Improve setting description for enabling A/B tests (#17585)
microsoft/vscode-python: news/2 Fixes/7793.md

> Improve setting description for enabling A/B tests. (Thanks [Thi Le](https://github.com/thi-lee))

Reviewer: Can you move this to `news/1 Enhancements` instead of `news/2 Fixes`?

Author: Done!

> Enabling A/B Testing description reworded (Thanks [Thi Le](https://github.com/thi-lee) for contributing)

Reviewer: ```suggestion
Enabling A/B Testing description reworded. (Thanks [Thi Le](https://github.com/thi-lee) for contributing)
```

> Enabling A/B Testing description reworded. (Thanks [Thi Le](https://github.com/thi-lee) for contributing)

Reviewer: ```suggestion
Improve setting description for enabling A/B tests. (Thanks [Thi Le](https://github.com/thi-lee) for contributing)
```

> Improve setting description for enabling A/B tests. (Thanks [Thi Le](https://github.com/thi-lee) for contributing)

Reviewer: ```suggestion
Improve setting description for enabling A/B tests. (Thanks [Thi Le](https://github.com/thi-lee))
```
